workshops/views.py

```diff
@@ -121,7 +129,10 @@ def get_context_data(self, **kwargs):
         else:
             context['title'] = 'New object'
 
-        context['form_helper'] = bootstrap_helper
+        form = context['form']
+        if not hasattr(form, 'helper'):
+            form.helper = BootstrapHelper(submit_label='Add')
```

Comment:
This is a default helper if no other is available, correct? You should add a comment about this.